### PR TITLE
Spanish Translations

### PR DESCRIPTION
**Pull request by free-beer**
This branch contains the changes related to introducing Spanish translations. Further changes will be made here.

### CODE REVIEW

File: languages/es.json [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+			"modifier": "Modificador de la tirada:"
+		},
+		"settings": {
+			"options": {
+				"randomizeHP": {
+					"blurb": "Indicates whether creatures added to scenes will have their HP randomly generated.",
+					"title": "Randomize Creature HP"
+				}
+			}
+		},
```
Comment (free-beer) [inline languages/es.json]: A few more translations needed here @WallaceMcGregor.

Comment (WallaceMcGregor) [reply languages/es.json]: "blurb": "Indica si las criaturas añadidas a las escenas tendrán sus PG generados aleatoriamente."
"title": "Aleatorizar PG de la criatura"

Comment (free-beer) [reply languages/es.json]: Ta. Got work out a few layout issues but I think I should have this out in a few days.

**Comment by WallaceMcGregor** [comment]
Thank you for your time and sorry for the problems I've made.

Now it's time to wait until you can make the modification of Black Sword Hack, which is what we have in the Spanish community translated.

Thank you very much for your time.

**Pull request merged**